src/lib/components/inventory/selected_item_info.ts

```diff
@@ -203,6 +230,23 @@ export class SelectedItemInfo extends FloatElement {
             } catch (e: any) {
                 console.error(`Failed to fetch float for ${this.asset.assetid}: ${e.toString()}`);
             }
+
+            // Fetch bluegem data if needed
+            if (this.itemInfo && isBlueSkin(this.itemInfo)) {
+                try {
+                    const bluegemResponse = await ClientSend(FetchBluegem, {
```

Comment:
nit: You can assign `this.bluegemData` directly from the response instead of this intermediate variable.